tests/i18n/test_extraction.py

```diff
@@ -687,6 +687,25 @@ def test_all_locales(self):
         self.assertTrue(os.path.exists(self.PO_FILE_KO))
         self.assertFalse(os.path.exists('locale/_do_not_pick/LC_MESSAGES/django.po'))
 
+    def test_locales_format_valid(self):
+        """Check the standard output(stdout) message in case the format of locale is valid"""
```

Comment:
These doc strings don't add much. I would delete them.

Reply:
Okay sure, will update it.